docs/release-notes/dracut_visible_warnings.rst

```diff
@@ -0,0 +1,11 @@
+:Type: Initial RAM disk (initrd)
+:Summary: Make installer critical warnings in Dracut/initrd more visible (#1983098)
+
+:Description:
+    Installation shows critical warnings raised in Dracut/initrd again when Anaconda is
+    starting or when Dracut starts to timeout. This should help users to resolve installation
```

Comment:
"starts to timeout" - maybe "times out" ?

Reply:
It's a bit tricky. The visible warnings will show up to user when retry attempts are half of the limit. So it still may be able to boot.

Reply:
Ah, then it's good as is, I think. It's hard to understand, but you don't want to write a novel to explain it...